
On i.MX27 (and probably i.MX21) bits per pixel selection in LCD controller
is slightly different than on i.MXL.
So makes some changes in MX1/2 framebuffer driver include...

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: linux-2.6.29/arch/arm/plat-mxc/include/mach/imxfb.h
===================================================================
--- linux-2.6.29.orig/arch/arm/plat-mxc/include/mach/imxfb.h	2009-05-27 20:07:19.000000000 +0200
+++ linux-2.6.29/arch/arm/plat-mxc/include/mach/imxfb.h	2009-05-27 21:09:24.000000000 +0200
@@ -13,7 +13,12 @@
 #define PCR_BPIX_4	(2 << 25)
 #define PCR_BPIX_8	(3 << 25)
 #define PCR_BPIX_12	(4 << 25)
+#ifdef CONFIG_ARCH_MX1
 #define PCR_BPIX_16	(4 << 25)
+#else
+#define PCR_BPIX_16	(5 << 25)
+#define PCR_BPIX_18	(6 << 25)
+#endif
 #define PCR_PIXPOL	(1 << 24)
 #define PCR_FLMPOL	(1 << 23)
 #define PCR_LPPOL	(1 << 22)
